### PR TITLE
[zh] Localize a case study: /bluecore/

### PR DESCRIPTION
**Pull request by windsonsea**
- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

https://istio.io/latest/about/case-studies/bluecore/

### CODE REVIEW

File: content/zh/about/case-studies/bluecore/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Bluecore 是一家多渠道个性化平台，专注于提供大规模的高度个性化的电子邮件、
+网站消息服务和付费媒体广告。由于 Bluecore 独特的产品需要处理客户推荐数据以及发送大量个性化的数字通信，
+他们需要处理大量数据的摄取、处理和发送工作。这个工作量巨大；
+在最近的黑色星期五促销活动期间，他们发送了超过 20 亿封电子邮件。
+
+![20 亿封电邮和 1.2 万个 Kubernetes Pod](./bluecore-statistics.png)
+
+除了数量之外，Bluecore 还需要速度。他们与大多数合作伙伴签订了服务协议，
+保证任何客户在提交工作后几个小时内收到电子邮件、网站消息或广告，
+这意味着处理速度至关重要。他们通过在 Google App Engine
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:29]:
```suggestion
这意味着处理速度至关重要。他们使用 Google App Engine
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+上运行的单体应用并运行了约 1.2 万个 Pod 的 Google Kubernetes Engine（GKE）
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:30]:
```suggestion
上运行的单体应用程序和运行约 12,000 个 Pod 的 Google Kubernetes Engine (GKE) 
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+集群来实现这一目标。
+
+## 挑战：单体架构和不断增长的数据流量 {#the-challenge-monlithic-architecture-and-increasing-data-traffic}
+
+Bluecore 是一个庞大的运营机构，对数据传输和处理有严格的要求。
+他们面临着一个艰巨的挑战：他们需要不断增加的数据量。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:36]:
```suggestion
他们面临着一个艰巨的挑战：他们需要处理的数据量不断增加。
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+他们知道如果自己的架构不能及时更新就无法处理不断增长的需求，整个流程管道将会过载。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:37]:
```suggestion
他们知道如果自己的架构不能及时更新就无法应对不断增长的需求，整个流程管道将会过载。
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+对他们的架构进行审查后发现，单体应用将是最大的挑战。
+Bluecore 的开发团队意识到，为了实现未来的增长，
+是时候开始迁移到更灵活和可扩展的基础架构了。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:41]:
```suggestion
是时候开始迁移到更灵活、可扩展的基础架构了。
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+Kubernetes 提供了一种扩展的路径，而且由于 Bluecore 已经在
+App Engine 上运行，将他们的工作流程的一部分迁移到 GKE
+似乎是一个明智的选择。然而，大多数 Bluecore
+的工程师没有足够的容器化应用经验。这可能使迁移到 GKE 变得困难。
+
+幸运的是，他们找到了一个解决方案。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:48]:
```suggestion
幸运的是，他们找到了解决方案。
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+## 解决方案：使用 Istio 服务网格赋能开发人员 {#the-solution-enabling-developers-with-the-istio-service-mesh}
+
+“Istio 使您能够立即开始构建”，Bluecore 基础设施团队的软件工程师
+Shray Kumar 解释说，“您可以确保以正确的方式构建一切。”
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:53]:
```suggestion
Shray Kumar 解释说，“而且你可以确保你正在以正确的方式构建。”
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+如果没有服务网格，就不会有明确的解决方案，
+将单体应用拆分为容器化服务会带来许多挑战。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:56]:
```suggestion
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+例如挑战之一是实现身份验证和鉴权。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:57]:
```suggestion
例如挑战之一是实现身份验证和授权。
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+的工程师没有足够的容器化应用经验。这可能使迁移到 GKE 变得困难。
+
+幸运的是，他们找到了一个解决方案。
+
+## 解决方案：使用 Istio 服务网格赋能开发人员 {#the-solution-enabling-developers-with-the-istio-service-mesh}
+
+“Istio 使您能够立即开始构建”，Bluecore 基础设施团队的软件工程师
+Shray Kumar 解释说，“您可以确保以正确的方式构建一切。”
+
+如果没有服务网格，就不会有明确的解决方案，
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:55]:
```suggestion
没有服务网格，将单体应用拆分为容易化服务就会带来许多没有明显解决方案的挑战。
```

Comment (hanxiaop) [reply content/zh/about/case-studies/bluecore/index.md:55]: 可能要拆分下

File: content/zh/about/case-studies/bluecore/index.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+将单体应用拆分为容器化服务会带来许多挑战。
+例如挑战之一是实现身份验证和鉴权。
+如果每个容器化服务都需要自己的解决方案，
+那么每个开发人员都很可能会采用自己的方式来解决。
+这可能导致代码碎片化并让后续的工作非常麻烦。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:60]:
```suggestion
这可能导致代码碎片化和未来很多麻烦的问题。
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+
+幸运的是，Kumar 和基础设施团队熟悉 Istio。
+他们与 Bluecore 数据平台团队的首席工程师
+Al Delucca 密切合作，制定了一个开始实现 Istio 的计划。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:64]:
```suggestion
Al Delucca 密切合作，制定了一个开始应用 Istio 的计划。
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+
+“我们遇到了这个问题”，Delucca 称，
+“但 Istio 是否是解决这个问题的合适工具，这是我们必须弄清楚的。”
+
+他们发现 Istio 的特性集提供了许多现有挑战的解决方案。
+鉴权是其中一大特性。来自合伙应用的传入消息需要进行身份验证。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:70]:
```suggestion
授权是其中一大特性。来自合伙应用的传入消息需要进行身份验证。
```

File: content/zh/about/case-studies/bluecore/index.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+Istio 可以在边缘执行这种身份验证，
+这意味着每个单独的服务不需要实现自己的方法。
+
+“我们能够将身份验证和鉴权推到边缘，这样我们的工程师就不需要理解这些系统的细节了”。
```
Comment (hanxiaop) [inline content/zh/about/case-studies/bluecore/index.md:74]:
```suggestion
“我们能够将鉴权和授权推到边缘，这样我们的工程师就不需要理解这些系统的细节了”。
```

File: content/zh/about/case-studies/bluecore/index.md (hunk 10 above)
Comment (windsonsea) [reply content/zh/about/case-studies/bluecore/index.md]: `容器化` 😺

File: content/zh/about/case-studies/bluecore/index.md (hunk 14 above)
Comment (windsonsea) [reply content/zh/about/case-studies/bluecore/index.md]: authentication and authorization，翻成`认证和授权`怎么样，参考 [istio authentication](https://istio.io/latest/zh/docs/tasks/security/authentication/)。

鉴权和授权应该都是 authorization (authn)

Comment (hanxiaop) [reply content/zh/about/case-studies/bluecore/index.md]: authn是鉴权，authz是授权。翻译成认证和授权感觉可以。

Comment (windsonsea) [reply content/zh/about/case-studies/bluecore/index.md]: ok，那先用认证和授权咯

**Review by wilsonwu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**